doc/classes/Basis.xml

```diff
@@ -340,94 +370,124 @@
 	</methods>
 	<members>
 		<member name="x" type="Vector3" setter="" getter="" default="Vector3(1, 0, 0)">
-			The basis matrix's X vector (column 0). Equivalent to array index [code]0[/code].
+			The basis' X axis, and the column [code]0[/code] of the matrix.
+			On an uniform basis, this vector points to the right ([constant Vector3.RIGHT]).
 		</member>
 		<member name="y" type="Vector3" setter="" getter="" default="Vector3(0, 1, 0)">
-			The basis matrix's Y vector (column 1). Equivalent to array index [code]1[/code].
+			The basis' Y axis, and the column [code]1[/code] of the matrix.
+			On an uniform basis, this vector points to the top ([constant Vector3.UP]).
 		</member>
 		<member name="z" type="Vector3" setter="" getter="" default="Vector3(0, 0, 1)">
-			The basis matrix's Z vector (column 2). Equivalent to array index [code]2[/code].
+			The basis' Z axis, and the column [code]2[/code] of the matrix.
+			On an uniform basis, this vector points to the back ([constant Vector3.BACK]).
 		</member>
 	</members>
 	<constants>
 		<constant name="IDENTITY" value="Basis(1, 0, 0, 0, 1, 0, 0, 0, 1)">
-			The identity basis, with no rotation or scaling applied.
+			The identity basis. This is a basis with no rotation, no shearing, and its scale being [code]1[/code]. This means that:
+			- The [member x] points to the right ([constant Vector3.RIGHT]);
+			- The [member y] points to the top ([constant Vector3.UP]);
+			- The [member z] points to the back ([constant Vector3.BACK]).
```

Comment:
You are mixing directions and sides here. Top is a side, back is a direction. We should use directions here, so top should be changed to up. If we were talking about sides then back would be changed to rear.